src/styles/signinPage.scss

```diff
@@ -0,0 +1,115 @@
+@import './_fonts.scss';
+@import './_variables.scss';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: montserrat, sans-serif;
+}
```

Comment:
Great work Samuel, can you try another way without using the selector all. It  can change our style

Reply:
thanks i can work on these